index.js

```diff
@@ -23,8 +28,7 @@ module.exports = function (input, length) {
 
 	while (i--) {
 		var x = pathParts[i];
-
-		if (remainingLength < x.length) {
+		if (remainingLength < x.length + 1) {
```

Comment:
The existing behaviour is already tested: https://github.com/sindresorhus/truncate-url/pull/3/files#r39253808